src/v/config/configuration.h

```diff
@@ -118,6 +118,7 @@ struct configuration final : public config_store {
     property<bool> enable_sasl;
     property<ss::sstring> static_scram_user;
     property<ss::sstring> static_scram_pass;
+    property<std::chrono::milliseconds> backend_housekeeping_interval_ms;
```

Comment:
what is `backend` i think you mean `coordinator_housekeeping...`

Reply:
the `cluster::controller_backed` is a class responsible for introducing changes in the cluster state.

Reply:
I know. I was asking to change the property name to something that makes sense rather than “backend”

Reply:
This property has more detailed description, I think that multiplying different names for the same entity makes things more confusing.